project.clj

```diff
@@ -7,10 +7,10 @@
 
   :pedantic? :abort
   :dependencies [[org.clojure/clojure "1.8.0"]
-                 [org.clojure/java.jdbc "0.7.5"]
+                 [org.clojure/java.jdbc "0.7.7"]
```

Comment:
This was required because Migratus updated itl.  The changes seem ok. https://github.com/clojure/java.jdbc/blob/java.jdbc-0.7.7/CHANGES.md